docs/docs/conceptual/using-gatsby-image.md

```diff
@@ -0,0 +1,83 @@
+
+### Gradual color transition
+
+In addition, when an image appears, it goes from blank background to fully there from one frame to another. This can also be visually jarring. Like css has a `transition` prop to help position shifts feel gradual, images feel more aesthetically pleasing when they have placeholders.
+
+Gatsby Image's will hold the spot for your image automatically when you specify `width` prop, and depending on your preference, will provide a background -- blurred, a background color, or traced SVG, while the image loads.
+
+![../images/gatsby-image-gif.gif]
+
+## Minimize image file size
+
+There are a number of ways to minimize image file size. Below are three of the most common, roughly in order of increasing complexity.
+
+### Cropping and compressing overly large images
+
+A common problem in larger projects is that images are uploaded into a CMS by content operations personnel by teams that may be less aware of appropriate web image sizes and formats. 
+
+For example, a support staff member may take a 1600x2000 pixel screenshot, save it as a PNG, and upload it alongside a helpdesk article. While this is a quite reasonable action, it may degrade page performance significantly. If the article has a 800px maximum width, a 640 x 800 pixel JPG would have displayed at the same quality but a tenth of the size; the extra weight may delay page load by a second or two. 
+
+You can solve this on your own via creating and maintaining a custom image processing pipeline, perhaps during CI/CD, to resize. However, that requires writing, and then maintaining, custom code.  Gatsby Image solves this out of the box; if you use the `width` prop, Gatsby Image will automatically resize larger underlying assets.
+
+### Generating "responsive images" for different device sizes
+
+Different devices have different screen sizes and resolutions, which means it may make sense to send smaller (or larger) objects. 
+
+Sending an image with a width of 800px to a  mobile phone with a 400px wide screen means that you are sending unnecessary data over the wire that will, again, delay page load, cause visitors to bounce, and hurt conversions. 
+
+Conversely, some laptops, like Macbook Pros, have a higher pixel-to-density ratio, so need "2x"-size images. 
+
+In order to support responsive images, you need to do the image processing beforehand, as well as generate the markup necessary. Gatsby Image has a `fluid` option that will transform images with a relatively straightforward API.
+
+### Better compression 
+
+The new [WebP image](https://developers.google.com/speed/webp) standard reduces image size by 25-35% for modern browsers. It's possible to support this standard, but the challenge is to also fall back for older browsers that don't support this.
```

Comment:
I wouldn't say it's a challenge (it's pretty simple), just that it needs to be considered.

Reply:
👍 good point